pybv/io.py

```diff
@@ -39,8 +40,10 @@ def write_brainvision(*, data, sfreq, ch_names, fname_base, folder_out,
     Parameters
     ----------
     data : ndarray, shape (n_channels, n_times)
-        The raw data to export. Data is assumed to be in **Volts** and will be
-        stored as specified by `unit`.
+        The raw data to export. Voltage data is assumed to be in
+        **Volts** and will be stored as specified by `unit`.
+        Other unsupported non-volt units are able to be specified
+        in ``unit`` parameter.
```

Comment:
```suggestion
        The raw data to export. Voltage data is assumed to be in **Volts** and
        will be scaled as specified by ``unit``. Non-voltage channels (as
        specified by ``unit``) are never scaled (e.g. '°C').
```

Reply:
Shouldn't this be 

    `'°C'`

(with backticks)

Reply:
I don't know, every project uses this differently. Backticks will render italic, whereas double backticks will render as monospace. Not sure what we want.